index.html

```diff
@@ -1107,38 +1190,187 @@ <h3>Supporting algorithms</h3>
+        mapping=][|source|].
+      </li>
+      <li>
+        If |platformCollector|'s [=platform collector/activated=] is false,
+        abort these steps.
+      </li>
+      <li>
+        In an [=implementation-defined=] manner, stop running the [=data
+        collection=] steps with |relevantGlobal|, |source|, and
+        |platformCollector|.
+      </li>
+      <li>
+        Set |platformCollector|'s [=platform collector/activated=] to false.
+      </li>
+      <li>
+        If |platformCollector|'s [=platform collector/associated pressure
+        source=] is a [=virtual pressure source=]:
+        <ol>
+          <li>
+            [=set/Remove=] |platformCollector| from its [=platform
+            collector/associated pressure source=]'s [=virtual pressure
+            source/connected platform collectors=].
+          </li>
+        </ol>
+      </li>
+      <li>
+        Otherwise, perform any [=implementation-defined=] steps to signal to
+        |platformCollector|'s [=platform collector/associated pressure source=]
+        to stop retrieving telemetry data.
+      </li>
+    </ol>
+    <p>
+      The <dfn>data collection</dfn> steps given |relevantGlobal|, |source| and
+      |platformCollector| are as follows:
+      <ol class="algorithm">
         <li>
-          Let |source:PressureSource| be the [=source type=] of the |data| sample.
+          Let |pressureSource| be |platformCollector|'s [=platform
+          collector/associated pressure source=].
         </li>
         <li>
-          Let |state:PressureState| be an [=adjusted pressure state=] given |data| and |source|.
+          If |pressureSource| is null, abort these steps.
         </li>
         <li>
-          Let |timestamp:DOMHighResTimeStamp| be a timestamp representing the time the |data| was
-          obtained from the |relevantGlobal|'s [=platform collector=].
-        <aside class="note">
-          The |data| sample and mapping between |data| sample, and [=pressure states=],
-          is [=implementation-defined=] and may use many different metrics. For instance,
-          for CPU, it might consider processor frequency and utilization, as well
-          as thermal conditions.
-        </aside>
+          Let |state:PressureState| be null.
+        </li>
+        <li>
+          If |pressureSource| is a [=virtual pressure source=]:
+          <ol>
+            <li>
+              Set |state| to |pressureSource|'s [=virtual pressure
+              source/pending sample=].
+            </li>
+            <li>
+              If |state| is null, abort these steps.
+            </li>
+          </ol>
+        </li>
+        <li>
+          Otherwise:
+          <ol>
+            <li>
+              Set |state| to an [=adjusted pressure state=] calculated in an
+              [=implementation-defined=] manner from |source| and any
+              [=implementation-defined=] platform data related to |source|.
```

Comment:
I think this is already described in the definition of "adjusted pressure state", right?

Reply:
That's a very good point. I've reverted commit 19e894ef88f4f096a931143498c53709faf5b686 and also hopefully made the part where we talk about platform collectors being able to connect to virtual pressure sources easier to understand.